builder/bundleset.go

```diff
@@ -27,6 +27,7 @@ type bundle struct {
 	OptionalIncludes []string
 	DirectPackages   map[string]bool
 	AllPackages      map[string]bool
+	AllRpmPackages   map[string]bool `json:"-"`
```

Comment:
what is the json tag used for here?

Reply:
we use this struct to create allbundles files. I dont want to include this field while writing that file

Reply:
Then keep it separate at the end of the structure with a note explaining it is not part of the file.
As an example, se how we do it in the config file:
https://github.com/clearlinux/mixer-tools/blob/7a776cbb665986d7215e7c7b14dd32c401701bb6/config/config.go#L33-L47